src/components/Designs.vue

```diff
@@ -0,0 +1,339 @@
+                      >
+                        <a
+                          :href="
+                            reactionLink(
+                              reactionKnockout,
+                              props.item.method,
+                              false
+                            )
+                          "
+                          class="link"
+                          target="_blank"
+                        >
+                          {{ reactionKnockout }}
+                        </a>
+                      </div>
+                    </div>
+                    <div
+                      v-if="props.item.design.reaction_knockouts.length > 10"
+                    >
+                      <a
+                        @click="showAllReactionKnockouts = true"
+                        :hidden="showAllReactionKnockouts"
+                      >
+                        ...
+                      </a>
+                    </div>
+                  </div>
+                </td>
+
+                <td align="rigth" width="15%">
+                  <div class="link-list">
+                    <div
+                      v-for="(geneKnockout, index) in props.item.design
+                        .gene_knockouts"
+                      :key="index"
+                    >
+                      <div v-if="index < 10">
+                        <a
+                          :href="geneLink(geneKnockout)"
+                          class="link"
+                          target="_blank"
+                        >
+                          {{ geneKnockout }}
+                        </a>
+                      </div>
+                      <div v-if="index >= 10" :hidden="!showAllGeneKnockouts">
+                        <a
+                          :href="geneLink(geneKnockout)"
+                          class="link"
+                          target="_blank"
+                        >
+                          {{ geneKnockout }}
+                        </a>
+                      </div>
+                    </div>
+                    <div v-if="props.item.design.gene_knockouts.length > 10">
+                      <a
+                        @click="showAllGeneKnockouts = true"
+                        :hidden="showAllGeneKnockouts"
+                      >
+                        ...
+                      </a>
+                    </div>
+                  </div>
+                </td>
+              </tr>
+            </template>
+          </v-data-table>
+        </div>
+      </v-flex>
+    </v-layout>
+  </v-container>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { mapGetters } from "vuex";
+
+export default Vue.extend({
+  name: "Designs",
+  data: () => ({
+    selected: [],
+    expand: true,
+    showAllReactionKnockins: false,
+    showAllReactionKnockouts: false,
+    showAllGeneKnockouts: false,
+    isDeletionDialogVisible: false,
+    isLoading: false,
+    headers: [
```

Comment:
Your column widths only add up to 95%, could that be a reason for layout problems when you expand?

Reply:
Oh, thank you, I should change it to 100%, initially I left 5% for checkbox area, but I couldn't specify width in the same way, expanded rows still need restyling though